services/datalad/datalad_service/tasks/publish.py

```diff
@@ -145,6 +149,43 @@ def publish_github_async(store, dataset, snapshot, github_remote):
     ds = store.get_dataset(dataset)
     publish_target(ds, github_remote, snapshot)
 
+def delete_s3_sibling(dataset_id, siblings, realm):
+    sibling = get_sibling_by_name(realm.s3_remote, siblings)
+    if sibling:
+        try:
+            session = boto3.Session(
+                aws_access_key_id=AWS_ACCESS_KEY_ID,
+                aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
+            )
```

Comment:
We'll need to make sure OpenNeuro's service accounts can do this, at the moment they cannot.

Reply:
Actually I'm not sure that we can grant the credentials to the same account, this seems to be an S3 limitation. We probably need to have a new set of environment variables for a second set of credentials.

Reply:
Went with client instead